### ISSUE TITLE
Import does not create the comment with the user name

### ISSUE THREAD

**Issue by wyrzykow** [BHTOM-Team/bhtom2#211]
Targets created via import, should also have the comment added with the name of the user who created it, similarly as in regular create. 
Example: https://bh-tom2.astrolabs.pl/targets/2327+4835/

**Comment by YuriiPurdenko** [comment]
https://github.com/BHTOM-Team/bhtom2/issues/196 the same issue, already on the prod

**Comment by YuriiPurdenko** [comment]
Please check this on production, and if everything is working as expected, feel free to close the issue.

**Comment by wyrzykow** [comment]
@YuriiPurdenko I tested it in prod on this:
name, GAIA_ALERTS_name, importance, cadence
Gaia24dhn,Gaia24dhn,1, 10
Gaia24dhh, Gaia24dhh, 4, 0.4
got errors:

<img width="988" alt="image" src="https://github.com/user-attachments/assets/80bac797-2d05-48fa-85a0-29c5c3054b81">

because there was no description. However, the Gaia broker should grab the description from Gaia. And that stopped working.  Added issue: https://github.com/BHTOM-Team/bhtom2/issues/224

When description field was added to the import: 
name, GAIA_ALERTS_name, importance, cadence, description
Gaia24dhn,Gaia24dhn,1, 10, FUOri candidate
Gaia24dhh, Gaia24dhh, 4, 0.4, CV candidate

The comment with the name of the creator was added correctly.
<img width="721" alt="image" src="https://github.com/user-attachments/assets/985ea78e-d042-4ecc-ba01-906da91af96b">

We can close this issue.
